client/integration-tests/bean-validation/src/test/java/io/quarkiverse/openapi/generator/it/BeanValidationTest.java

```diff
@@ -43,9 +44,10 @@ void testValidationAnnotationsAreInPlaceApi() {
     void testValidationAnnotationsAreInPlaceModel() throws Exception {
         Field id = ValidatedObject.class.getDeclaredField("id");
         Field name = ValidatedObject.class.getDeclaredField("name");
+        Field secondName = ValidatedObject.class.getDeclaredField("secondName");
         Field size = ValidatedObject.class.getDeclaredField("size");
 
-        assertThat(Arrays.stream(ValidatedObject.class.getFields())
+        assertThat(Stream.of(id, name, secondName, size)
```

Comment:
Just to be sure, have you changed this only for performance reasons?

Reply:
Actually not.

`getFields` method returns only `public` fields and the generated class consists only of private ones. It makes `ValidatedObject.class.getFields()` always returning an empty array.
By this reason assertion `assertThat(Arrays.stream(ValidatedObject.class.getFields())` has no chance to fail.
`getDeclaredFields()` method would be more appropriate, but I decided to utilize the objects returned by `getField` a few lines above.

I've just added the same change in the second test method.